camayoc/db_snapshot.py

```diff
@@ -127,3 +134,21 @@ def _connection_job_itemgetter(item):
     credential_id = item.get("credential", {}).get("id", 0)
     name = item.get("name", "")
     return f"{name}-{credential_id}-{source_id}"
+
+
+def _source_fact_itemgetter(item):
+    vm_uuid = item.get("vm.uuid", "")
+    vm_host_uuid = item.get("vm.host.uuid", "")
+    if vm_uuid or vm_host_uuid:
```

Comment:
@quipucords/development do you know if either / both are guaranteed to have a value assigned by vCenter?

Reply:
Unfortunately, I think not, but I cannot say with confidence because I believe the related inspection/fingerprinting logic was based on reverse-engineering whatever data we saw at the time from vcenter.

Reply:
Thanks. In that case I will leave the `or` and hope that at least one of them is always available. Testlab data seems to have both filled for all hosts, so this has no immediate practical implications.